src/VSIXExpInstaller/VSIXExpInstaller/Program.cs

```diff
@@ -49,14 +57,15 @@ private static bool FindArg(List<string> args, string argName)
 
         private static void PrintUsage()
         {
-            Console.WriteLine("Usage: VSIXExpInstaller [/rootSuffix:suffix] [/u] [/uninstallAll] [/vsInstallDir:vsInstallDir] <vsix>");
+            Console.WriteLine("Usage: VSIXExpInstaller [/rootSuffix:suffix] [/skipIfEqualOrNewerInstalled] [/u] [/uninstallAll] [/vsInstallDir:vsInstallDir] <vsix>");
             Console.WriteLine("Suffix is Exp by default.");
         }
 
         private static string GetDevenvPath(string vsInstallDir)
         {
             if (string.IsNullOrWhiteSpace(vsInstallDir))
             {
+                Environment.ExitCode = E_ACCESSDENIED;
```

Comment:
nit: I thought you could set the ErrorCode of an exception and it would do the same...